src/compas_robots/viewer/scene/__init__.py

```diff
@@ -0,0 +1,16 @@
+from compas.plugins import plugin
+from compas.scene import register
+
+from compas_robots import RobotModel
+
+from .robotmodelobject import RobotModelObject
+
+
+@plugin(category="factories", requires=["compas_viewer"])
+def register_scene_objects():
+    register(RobotModel, RobotModelObject, context="Viewer")
```

Comment:
Correct me if I'm wrong, but I think the build fails because it's requiring `compas_viewer` to be installed, which shouldn't be the case. If you move the import of `from .robotmodelobject import RobotModelObject` inside the `register_scene_objects` function, it will not try to import if the `requires` is not fulfilled.